Makefile

```diff
@@ -60,3 +74,26 @@ devrel:
 devrelease:
 	$(REBAR) as dev release
 
+grpc:
+	@echo "generating miner grpc services"
+	REBAR_CONFIG="config/grpc_client_gen.config" $(REBAR) grpc gen
```

Comment:
prob a good idea to rename this based on the service it supports, something like grpc_client_gen_local.config

I will do the same on the POC branch for the gateway service.  that way we avoid any conflict,  allows each service to have its own config file with its own gpb options etc

Reply:
oof; i had just figured they'd merge together but yeah, if you need different gpb options that will get messy; will do.